demosplan/DemosPlanCoreBundle/ResourceTypes/AdministratableUserResourceType.php

```diff
@@ -77,6 +79,8 @@ protected function getAccessConditions(): array
             $this->conditionFactory->propertyHasValue(false, $this->deleted),
             // never show internal Citizen user
             $this->conditionFactory->propertyHasNotValue(User::ANONYMOUS_USER_ID, $this->id),
+            // never show ApiAiUser
+            $this->conditionFactory->propertyHasNotValue( AiApiUser::AI_API_USER_ID, $this->id),
```

Comment:
Could you please use AI_API_USER_LOGIN? This should be more stable

Reply:
done in [ad4f9a1](https://github.com/demos-europe/demosplan-core/pull/2705/commits/ad4f9a165ea864b05680491f786cbe5a862a83ed)